### PR TITLE
rewritten YamlService.cs with YamlDotNet yaml serialization and questionable anonymous type objects.

### PR DESCRIPTION
**Pull request by tefarley**

### CODE REVIEW

File: Hippo.Web/Services/YamlService.cs [hunk 1]
```diff
@@ -26,24 +27,32 @@ public async Task<string> Get(User currentUser, AccountCreateModel accountCreate
...
+                        cluster = sponsorAccount.Cluster
+                    },
+                    account = new
+                    {
+                        name = currentUser.Name,
+                        email = currentUser.Email,
+                        kerb = currentUser.Kerberos,
+                        iam = currentUser.Iam,
+                        mothra = currentUser.MothraId,
+                        key = accountCreateModel.SshKey
```
Comment (jSylvestre) [inline Hippo.Web/Services/YamlService.cs:52]: Does it need the extra quotes?

Comment (tefarley) [reply Hippo.Web/Services/YamlService.cs:52]: pardon? I'm not sure I understand

Comment (jSylvestre) [reply Hippo.Web/Services/YamlService.cs:52]: The code that was deleted looked like this
sb.AppendLine($"    key: \"{accountCreateModel.SshKey}\"");

With escaped double quotes. I don't remember why we did that originally.

Comment (tefarley) [reply Hippo.Web/Services/YamlService.cs:52]: Ah I see. You could be right, something else may expect the quotes. Thank you. Lemme check with Omen to be sure it's not a requirement on the puppet side. I'll see if I can do a more in-depth test on my local hippo. If anything I can just add the quotes back in the new version

**Review by jSylvestre — APPROVED** [review]

File: Hippo.Web/Services/YamlService.cs (hunk 1 above)
Comment (camillescottatwork) [reply Hippo.Web/Services/YamlService.cs:52]: @jSylvestre see #189: you don't want to be in the business of formatting the YAML yourself; the serializer will handle escaping and quoting as-needed automagically.

File: Hippo.Web/Hippo.Web.csproj [hunk 2]
```diff
@@ -21,11 +21,11 @@
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.2" />
-    <PackageReference Include="MvcReact" Version="1.0.1" />
```
Comment (jSylvestre) [inline Hippo.Web/Hippo.Web.csproj:24]: You probably need to add this back.

**Review by sprucely — CHANGES_REQUESTED** [review]

File: Hippo.Web/Hippo.Web.csproj (hunk 2 above)
[the remark this answers is not in the document]
Comment (tefarley) [reply Hippo.Web/Hippo.Web.csproj:24]: Ah that was an accident thank you.

**Review by sprucely — APPROVED** [review]

**Pull request merged**